lms/djangoapps/learner_dashboard/views.py

```diff
@@ -66,6 +67,7 @@ def program_details(request, program_uuid):
 
     context = {
         'program_data': program_data,
+        'courses_progress': meter.progress(programs=meter.programs, count_only=False)[0],
```

Comment:
There's going to be a lot duplicated between `program_data` and `courses_progress`. The bulk of `program_data` is data about the courses in the program, and it looks like that same data is going to end up grouped in `course_progress`. What can we do about that?

I'd update the tests of this view so that they verify the data is present on the page. Also, if these end up staying separate, a more descriptive name might be nice here. `course_groups`, for example.